Payload_Type/Apollo/agent_code/Apollo/Mythic.cs

```diff
@@ -301,6 +306,16 @@ public struct SocksDatagram
             public string data;
         }
 
+        public struct PortFwdDatagram
+        {
+            public Dictionary<String,Dictionary<String,Dictionary<String,Dictionary<String,Dictionary<int,String>>>>> data;
```

Comment:
Same comment as earlier - this should probably be some form of nested structure so this is easier to grok